### ISSUE TITLE
Sequence diagram image not rendered in display feature section of DG

### ISSUE THREAD

**Issue by rachelyeohm** [rachelyeohm/ped#5]
![image.png](https://raw.githubusercontent.com/rachelyeohm/ped/main/files/1108d1c5-552f-41ba-8f3c-fd1ea3e2ad63.png)
Based on the alt text and the broken image icon, a sequence diagram image is likely to be shown - however, it is missing. It would be good to ensure the image is added correctly.

A possible reason for this error: the puml file name you were referencing in the DG code does not match with the puml file name of the sequence diagram.
